<launch>
	<arg name="tello_ip" default="192.168.10.1" />
	<arg name="namespace"			default="tello" />

	<group ns="$(arg namespace)">

		<node pkg="tello_pilot" name="tello_pilot_node" type="tello_pilot_node.py" output="screen">
            <!-- camer_fps [Hz]: This can be 5, 15, or 30. --> 
			<param name="camera_fps" value="30" type="int" /> 
            <!-- camera_bitrate [MB/s]: This can be auto, 1, 2, 3, 4, or 5. --> 
            <param name="camera_bitrate" value="auto" type="str" />
					
		</node>

	</group>

</launch>
